web/perma_payments/tests/test_routes.py

```diff
@@ -118,7 +118,8 @@ def subscribe():
             'recurring_frequency': SENTINEL['recurring_frequency'],
             'recurring_start_date': SENTINEL['date'],
             'link_limit': SENTINEL['link_limit'],
-            'link_limit_effective_timestamp': SENTINEL['datetime'].timestamp()
+            'link_limit_effective_timestamp': SENTINEL['datetime'].timestamp(),
+            'reference_number': SENTINEL['reference_number']
```

Comment:
I think reference number doesn't need to be in this fixture, which is only used for the "make me a new subscription" route.